src/main/java/com/minecolonies/api/colony/requestsystem/requestable/Tool.java

```diff
@@ -189,16 +191,16 @@ public boolean matches(@NotNull final ItemStack stack)
                                          && stack.getCount() >= 1
                                          && getToolClasses(stack).stream()
                                               .filter(s -> getToolClass().getName().equalsIgnoreCase(s))
-                                              .map(ToolType::getToolType)
-                                              .anyMatch(t -> t != ToolType.NONE && (stack.getDamageValue() > 0 || !stack.isDamaged()) && ItemStackUtils.hasToolLevel(stack,
+                                              .map(ToolTypeEntry::getToolType)
+                                              .anyMatch(t -> t != ModToolTypes.none.get() && (stack.getDamageValue() > 0 || !stack.isDamaged()) && ItemStackUtils.hasToolLevel(stack,
                                                 t,
                                                 getMinLevel(),
                                                 getMaxLevel()));
 
         if (!toolTypeResult)
```

Comment:
Pretty sure this block is unnecessary given that you now iterate all tool types in the registry, including shields and flint and steel, so this extra if should not be necessary.

Reply:
Done.